### ISSUE TITLE
cli: --skip flag not working with YAML syntax

### ISSUE THREAD

**Issue by vipulchhabra99** [getpopper/popper#795]
The skip option doesn't seem to work with YAML files. I am working with this sample YAML file
```
version: '1'
steps:
- id: "a1"
  uses: "sh"
  runs: "ls"

- id: "a2"
  needs: "a1"
  uses: "sh"
  runs: "ls"

- id: "b"
  needs: ["a1","a2"]
  uses: "sh"
  runs: "ls"

- id: "c"
  needs: ["a1","a2"]
  uses: "sh"
  runs: "ls"

- id: "d"
  needs: ["b","c"]
  uses: "sh"
  runs: "ls"

```
On running `popper run --dry-run -f test.yml --skip a2`
Its provides similar output to `popper run --dry-run -f test.yml`

**Comment by JayjeetAtGithub** [comment]
@vipulchhabra99  Thanks for reporting. I will look into this.

**Comment by ivotron** [comment]
I pushed a commit for hidding these arguments for now until we resolve this

**Comment by ivotron** [comment]
fixed in 8d3b0a25829482b09d15a446b7ab7c76bf58f96a
